Dockerfile.slim

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2017-present Sonatype, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5-204
+
+# Build parameters
+ARG IQ_SERVER_VERSION=1.135.0-01
+ARG IQ_SERVER_SHA256=5005ba518380ae3dc70f09de5cc43b9e70e7eff979d1fb9ad07ff35985febfdf
+
+
+ARG TEMP="/tmp/work"
+ARG IQ_HOME="/opt/sonatype/nexus-iq-server"
+ARG SONATYPE_WORK="/sonatype-work"
+ARG CONFIG_HOME="/etc/nexus-iq-server"
+ARG LOGS_HOME="/var/log/nexus-iq-server"
+ARG GID=1000
+ARG UID=1000
+ARG TIMEOUT=600
+
+LABEL name="Nexus IQ Server image" \
+  maintainer="Sonatype <support@sonatype.com>" \
+  vendor=Sonatype \
+  version="${IQ_SERVER_VERSION}" \
+  release="1.135.0" \
+  url="https://www.sonatype.com" \
+  summary="The Nexus IQ Server" \
+  description="Nexus IQ Server is a policy engine powered by precise intelligence on open source components. \
+    It provides a number of tools to improve component usage in your software supply chain, allowing you to \
+    automate your processes and achieve accelerated speed to delivery while also increasing product quality" \
+  com.sonatype.license="Apache License, Version 2.0" \
+  com.sonatype.name="Nexus IQ Server image" \
+  run="docker run -d -p 8070:8070 -p 8071:8071 IMAGE" \
+  io.k8s.description="Nexus IQ Server is a policy engine powered by precise intelligence on open source components. \
+    It provides a number of tools to improve component usage in your software supply chain, allowing you to \
+    automate your processes and achieve accelerated speed to delivery while also increasing product quality" \
+  io.k8s.display-name="Nexus IQ Server" \
+  io.openshift.expose-services="8071:8071" \
+  io.openshift.tags="Sonatype,Nexus,IQ Server"
+
+USER root
+
+# For testing
+RUN microdnf update -y \
+&& microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y java-1.8.0-openjdk-devel \
+&& microdnf install -y procps gzip unzip tar shadow-utils findutils util-linux less rsync \
```

Comment:
this Dockerfile is a copy of the original Dockerfile, but leaves out `git` here.
Compare to https://github.com/sonatype/docker-nexus-iq-server/blob/INT-6717-alternative-slim-image/Dockerfile#L56